lenovo/thinkpad/p1/default.nix

```diff
@@ -9,9 +9,12 @@
   # Need to set Thunderbolt to "BIOS Assist Mode"
```

Comment:
It seems that including nvidea above breaks evaluation since the bus ID is not configured:

```
$ ./tests/run.py '<nixos-hardware/lenovo/thinkpad/p1>'
FAIL <nixos-hardware/lenovo/thinkpad/p1>:
warning: unknown setting 'access-tokens'
warning: unknown setting 'experimental-features'
warning: unknown setting 'flake-registry'
error: while evaluating anonymous function at /home/joerg/git/nixos-hardware/tests/build-profile.nix:1:1, called from undefined position:
while evaluating the attribute 'system' at /home/joerg/git/nixpkgs/nixos/default.nix:38:3:
while evaluating the attribute 'config.system.build.toplevel' at /home/joerg/git/nixpkgs/nixos/modules/system/activation/top-level.nix:293:5:
while evaluating 'foldr' at /home/joerg/git/nixpkgs/lib/lists.nix:52:20, called from /home/joerg/git/nixpkgs/nixos/modules/system/activation/top-level.nix:128:12:
while evaluating 'fold'' at /home/joerg/git/nixpkgs/lib/lists.nix:55:15, called from /home/joerg/git/nixpkgs/lib/lists.nix:59:8:

Failed assertions:
- When NVIDIA PRIME is enabled, the GPU bus IDs must configured.

The following 1 test(s) failed:
./tests/run.py '<nixos-hardware/lenovo/thinkpad/p1>'
```

Do you see any way to fix this? Otherwise I would remove the nvidia module again and only include in models where we no the nvidia bus ids.